src/test/java/com/ixortalk/organization/api/rest/OrganizationRestController_DeleteCascade_WithoutAssetMgmt_IntegrationAndRestDocTest.java

```diff
@@ -0,0 +1,106 @@
+import static org.apache.http.HttpStatus.SC_NO_CONTENT;
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+
+@TestPropertySource(properties = {"ixortalk.server.assetmgmt.url=false"})
+public class OrganizationRestController_DeleteCascade_WithoutAssetMgmt_IntegrationAndRestDocTest extends AbstractSpringIntegrationTest {
+
+    public static final AssetId FIRST_DEVICE_ASSET_ID = assetId(nextString("firstDeviceAssetId"));
+    public static final AssetId SECOND_DEVICE_ASSET_ID = assetId(nextString("secondDeviceAssetId"));
+
+    @Before
+    public void before() throws JsonProcessingException {
+
+        assetMgmtWireMockRule.stubFor(
+                post(urlEqualTo("/assetmgmt/assets/search/property"))
+                        .andMatching(retrievedAdminTokenAuthorizationHeader())
+                        .withHeader(CONTENT_TYPE, equalTo(APPLICATION_JSON_VALUE))
+                        .withRequestBody(equalToJson(objectMapper.writeValueAsString(organizationX.getOrganizationId())))
+                        .willReturn(okJson(objectMapper.writeValueAsString(newArrayList(
+                                AssetTestBuilder.anAsset().withAssetId(FIRST_DEVICE_ASSET_ID).build(),
+                                AssetTestBuilder.anAsset().withAssetId(SECOND_DEVICE_ASSET_ID).build()
+                        )))));
+
+        assetMgmtWireMockRule.stubFor(
+                put(urlEqualTo("/assetmgmt/assets/" + FIRST_DEVICE_ASSET_ID.stringValue() + "/properties"))
+                        .andMatching(retrievedAdminTokenAuthorizationHeader())
+                        .withRequestBody(equalToJson(objectMapper.writeValueAsString(assetMgmtFieldsToBeCleared())))
+                        .willReturn(ok()));
+        assetMgmtWireMockRule.stubFor(
+                put(urlEqualTo("/assetmgmt/assets/" + SECOND_DEVICE_ASSET_ID.stringValue() + "/properties"))
+                        .andMatching(retrievedAdminTokenAuthorizationHeader())
+                        .withRequestBody(equalToJson(objectMapper.writeValueAsString(assetMgmtFieldsToBeCleared())))
+                        .willReturn(ok()));
+
+        organizationCallbackApiWireMockRule.stubFor(post(urlPathEqualTo("/org-callback-api" + ORGANIZATION_REMOVED_CALLBACK_PATH.configValue()))
+                .withQueryParam("organizationId", equalTo(valueOf(organizationX.getId())))
+                .andMatching(retrievedAdminTokenAuthorizationHeader())
+                .willReturn(ok()));
+    }
+
+
+    @Test
+    public void assetMgmtNotCalledOnCascadedDelete() throws JsonProcessingException {
+
+        given()
+                .auth()
+                .preemptive()
+                .oauth2(ADMIN_JWT_TOKEN)
+                .when()
+                .delete("/organizations/{id}/cascade", organizationX.getId())
+                .then()
+                .statusCode(SC_NO_CONTENT);
+
+        assetMgmtWireMockRule.verify(0,
```

Comment:
We should verify that no calls happen at all, not only two very specific calls.  We also need to test again that everything else does happen (removing organization itself, performing callbacks, .... ).